src/components/postDetail/ReviewList/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import classNames from 'classnames/bind';
+
+import { REVIEW_SORT_OPTIONS } from '@/constants';
+import { getReviewPageSize } from '@/utils/getPageSize';
+
+import Dropdown from '@/components/commons/Dropdown';
+import Pagination from '@/components/commons/Pagination';
+import ReviewCard from '@/components/commons/ReviewCard';
+import ReviewSummary from '@/components/postDetail/ReviewSummary';
+import { REVIEW_LIST_DATA } from '@/constants/mockData/reviewList';
+import { useDeviceType } from '@/hooks/useDeviceType';
+import usePaginatedDataList from '@/hooks/usePaginatedDataList';
+import useSortedDataList from '@/hooks/useSortedDataList';
+
+import { Order, Review, ReviewResponse, SortOption } from '@/types';
+
+import styles from './ReviewList.module.scss';
+
+const cx = classNames.bind(styles);
+
+type ReviewListProps = {
+  list: ReviewResponse;
+};
+
+const ReviewList = ({ list = REVIEW_LIST_DATA }: ReviewListProps) => {
+  const [page, setPage] = useState(1);
+
+  const currentDeviceType = useDeviceType();
+  const pageSize = getReviewPageSize(currentDeviceType);
+
+  const initialDataList: Review[] = list.reviews;
```

Comment:
```suggestion
  const { totalCount, reviews: initialDataList }: ReviewResponse = list;
```

요렇게 쓰면 아래에서 totalCount 를 사용할 때도 `list.` 을 안붙이고 사용 가능할것 같습니다!